src/screens/ImportSeedPhraseSheet.js

```diff
@@ -1,27 +1,32 @@
 import analytics from '@segment/analytics-react-native';
 import { isValidAddress } from 'ethereumjs-util';
+import { isEmpty as isObjectEmpty } from 'lodash';
```

Comment:
Can we use just isEmpty instead of isObjectEmpty as there are no naming conflicts with isEmpty

Reply:
Actually, given below comment for line 257, we can get rid of isEmpty and just bring in filter